### DISCUSSION TITLE
Renovate does not detect maven-wrapper

### DISCUSSION THREAD

**Discussion by bmarwell** [renovatebot/renovate#16034] [Suggest an Idea]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

maven-wrapper writes the following files:

* `mvnw`
* `mvnw.cmd`
* `.mvn/wrapper/maven-wrapper.properties`

Those files need to be updated.

Suggestion: Do this in two or three steps.

1. updates in maven-wrapper.properties
 * version numbers for `distributionUrl` (maven itself)
 * version number for wrapper (`wrapperUrl`).
3. update scripts using `./mvnw org.apache.maven.plugins:maven-wrapper-plugin:3.1.1:wrapper -Dmaven=3.8.6` (where 3.1.1 is the wrapper distribution and plugin version and 3.8.6 is the maven version).

Example repo:
Any with maven wrapper, e.g. https://github.com/bmarwell/cowsay-liberty-feature/issues/1

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment]


**Reply by bmarwell** [reply]
Sorry, but I do know the difference. Yes, I see why you see this as a feature request instead (it was not INTENDED to work... yet). For me it seemed like a bug (it misses something although it should not have), there's not always a clear border between those. For me it was a 50:50 decision whether to file this as a bug or feature request. Sorry I picked the wrong one in your opinion.

BUT...

> If you don't know the difference between a bug and a feature

... this really does not sound nice. You may want to rephrase that.

**Reply by rarkins** [reply]
If this is what you truly believe, please do not create ANY future issues in this repo (neither features nor bugs) and stick to discussions only.

**Reply by bmarwell** [reply]
Look. I now know it wasn't intended to work. Kindly retag this as an issue then.

**Reply by rarkins** [reply]
A feature request already exists: #3566. So too does a Pull Request: #15660, so you are welcome to check that out.

**Reply by bmarwell** [reply]
Thanks for pointing me there.
